src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -342,8 +342,9 @@ public function request($method, $uri, array $parameters = array(), array $files
     protected function doRequestInProcess($request)
     {
         $deprecationsFile = tempnam(sys_get_temp_dir(), 'deprec');
-        putenv('SYMFONY_DEPRECATIONS_SERIALIZE='.$deprecationsFile);
-        $process = new PhpProcess($this->getScript($request), null, null);
+        $environment = getenv();
```

Comment:
this doesn't work before PHP 7.1
but I don't get what it changes in fact, so maybe it's not required at all?

Reply:
What this was trying to do was only set SYMFONY_DEPRECATIONS_SERIALIZE in the sub process so we didn't have to unset it after making the request.

Reply:
why not just relying on the feature of the Process component inheriting the env variables ?